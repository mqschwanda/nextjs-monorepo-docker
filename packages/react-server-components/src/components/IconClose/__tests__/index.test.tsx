import { render } from '@testing-library/react';
import { IconClose } from '@';

describe('@mqs/react-server-components', () => {
  describe('components', () => {
    describe('<IconClose />', () => {
      it('renders', () => {
        const testId = IconClose.name;

        const { getByTestId, asFragment } = render((
          <IconClose
            className='test'
            testId={testId}
          />
        ));

        const element = getByTestId(testId);
        expect(element).toBeTruthy();

        expect(asFragment()).toMatchSnapshot();
      });
    });
  });
});
