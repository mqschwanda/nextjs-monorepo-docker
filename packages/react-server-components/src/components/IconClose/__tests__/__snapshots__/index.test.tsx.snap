// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@mqs/react-server-components components <IconClose /> renders 1`] = `
<DocumentFragment>
  <svg
    class="stroke-current test"
    data-testid="IconClose"
    fill="none"
    stroke="currentColor"
    viewBox="0 0 24 24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M6 18L18 6M6 6l12 12"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </svg>
</DocumentFragment>
`;
